robottelo/common/helpers.py

```diff
@@ -151,6 +151,32 @@ def generate_strings_list(len1=None):
     ]
 
 
+def generate_strings_dict(len1=None):
+    """Generates a dictionary of all the input strings.
+
+    :param int len1: Specifies the length of the strings to be
+        be generated. If the len1 is None then the dict is
+        returned with string types of random length.
+    :return: Returns a dict of various string types.
+    :rtype: dictionary
+
+    """
+    if len1 is None:
+        len1 = gen_integer(3, 30)
+    return {
+        str_type: gen_string(str_type, len1)
+        for str_type
+        in (u'alpha', u'numeric', u'alphanumeric',
+            u'latin1', u'utf8', u'cjk', u'html')
+    }
+
+
+def removekey(d, key):
+    r = dict(d)
+    del r[key]
+    return list(r.values())
```

Comment:
Yes, but the idea is in future any tests  module could make use of it, for removing a particular data, when we encounter the below scenario.

Neither 'skipping the entire testcase' is desired nor 'adding a bug to numeric data' as there will be many testcases and would add up extra lines of code to the existing tests